helm/microcosm/values.yaml

```diff
@@ -155,6 +155,10 @@ ingestion:
         limits:
           memory: '2Gi'
           cpu: '2'
+      args:
+        CHANGESET_TIMEOUT_OPEN: 1 day
+        CHANGESET_TIMEOUT_IDLE: 1 day
+        MAX_PAYLOAD: "5000000"
```

Comment:
This should also be copied to the `myvalues.example.yaml` file.

Reply:
Done